packages/polar/src/lib/deploy/contract.ts

```diff
@@ -25,7 +25,7 @@ import type {
   UserAccount
 } from "../../types";
 import { loadCheckpoint, persistCheckpoint } from "../checkpoints";
-import { ExecuteResult, getClient, getSigningClient } from "../client";
+import { getClient, getSigningClient } from "../client";
```

Comment:
Do we have an alternative to `ExecuteResult` here? Cause using `any` is not ideal.

Reply:
we don't

Reply:
no we don't

Reply:
maybe we can run this, see the value types and create a new type in type.ts
For that we need schema and cosmwasm 1.0 support.